### DISCUSSION TITLE
Autoriser automatiquement la réception et l'envoi de fichiers

### DISCUSSION THREAD

**Discussion by thefrenchlawyer** [localsend/localsend#1280] [L-Français]
Bonjour,

Je viens d'abandonner Nitroshare et ai rejoint LocalSend qui m'a séduit par son UI et sa simplicité. J'ai pu faire mes premiers transferts.

J'aimerais savoir s'il est possible d'autoriser à l'avance les transferts dans quelques sens que ce soit (en transfert ou en arrivée). A chaque envoi, la réception doit être validée sur l'ordinateur de destination, par une fenêtre pop-up.

Ce réglage existe-t-il sur Windows, et sur Android ?

J'ai recherché dans le forum Anglais/International, s'il n'existait pas une discussion qui traitait du sujet, et sauf erreur de ma part, je n'ai pas trouvé de réponse à ma question. J'ai également parcouru les paramètres de l'application (Windows/Android), et n'y ai pas trouvé le réglage adéqaut.

Vous remerciant à tous pour vos réponses

Bonne journée

**Comment by Caesarovich** [comment]
Bonjour @thefrenchlawyer ,

L'option "Quick save" est ce que vous recherchez je pense.


Vous pouvez la trouver ici:

![Screenshot from 2024-06-01 18-29-12](https://github.com/localsend/localsend/assets/38408878/2917cece-9ca3-4a77-813e-4db26ca9a281)
